src/templates/rest-api/tests/docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: '3.9'
+services:
+  test-create-project:
+    build:
+      context: ../
+      dockerfile: ./tests/Dockerfile
+    volumes:
+      - ../:/pytest_project
```

Comment:
Haven't seen this colon `/:/` in the path . Does it have a name so I can learn what it does? Thanks

Reply:
it is just a separation between two mounted volumes.  Here I use this spelling just because I want to mount one step back

Reply:
So the same as

```
volumes:
      - ../
      - /pytest_project
 ```